src/index.ts

```diff
@@ -69,6 +69,15 @@ async function main() {
 	}
 
 	await startAPIServer();
+
+	if (config.maxRunTime) {
+		setTimeout(async () => {
+			logger.info(
+				`We've been searching for over ${config.maxRunTime} seconds as configured by MAX_RUNTIME. Time to call it quits for now. Gracefully stopping program.`
+			);
+			await stopAndExit();
+		}, config.maxRunTime);
```

Comment:
This is in milliseconds, so you will have to convert.

Reply:
Doh! Thanks for catching.